tests/test_download.py

```diff
@@ -168,44 +170,29 @@ def test_download_url_to_tempfileobj_and_urls(self):
     self.assertRaises(securesystemslib.exceptions.FormatError,
                       download_file, None, self.target_data_length)
 
-    self.assertRaises(securesystemslib.exceptions.FormatError,
+    self.assertRaises(requests.exceptions.MissingSchema,
```

Comment:
Again, not related to this PR, but moving to `assertRaises` as a context processor would make this code more readable...